packages/core/src/Cardano/types/Script.ts

```diff
@@ -226,3 +228,26 @@ export interface PlutusScript {
  * Program that decides whether the transaction that spends the output is authorized to do so.
  */
 export type Script = NativeScript | PlutusScript;
+
+/**
+ * Gets the policy id of the given native script.
+ *
+ * @param script The native script to get the policy ID of.
+ * @returns The policy Id.
+ */
+export const nativeScriptPolicyId = (script: NativeScript): Cardano.PolicyId =>
+  Cardano.PolicyId(Buffer.from(nativeScript(script).hash().to_bytes()).toString('hex'));
+
+/**
+ * Gets the public key hash from the given Bech32 address.
+ *
+ * @param address The address to get the public key hash from.
+ * @returns the key hash.
+ */
+// TODO: Use opaque strings
+export const policyKeyHash = (address: Cardano.Address): string => {
```

Comment:
Isn't this just [Ed25519KeyHash](https://github.com/input-output-hk/cardano-js-sdk/blob/e4197cdae87991abc7565946f50a9d82189ab7ec/packages/core/src/Cardano/types/Key.ts#L46)?

Also, this util is likely not needed. See comment below.

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/commit/96640f3b9db0271f290550eeb86a2f9809ee9358